src/simpred.c

```diff
@@ -108,7 +108,7 @@ int main(int argc, char const *argv[])
 void desaloca(BPB * buffer){
 	for (int i = buffer->size - 1; i >=0 ; --i)
 	{
-        if(buffer != NULL)
+        if(buffer)
```

Comment:
Não poderia usar um operador ternário?

Reply:
```suggestion
        if(buffer)
```
Sim seria melhor para diminuir a complexidade do codigo